x-pack/plugin/inference/src/test/java/org/elasticsearch/xpack/inference/services/openshiftai/request/rarank/OpenShiftAIRerankRequestEntityTests.java

```diff
@@ -0,0 +1,63 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+package org.elasticsearch.xpack.inference.services.openshiftai.request.rarank;
+
+import org.elasticsearch.common.Strings;
+import org.elasticsearch.test.ESTestCase;
+import org.elasticsearch.xcontent.ToXContent;
+import org.elasticsearch.xcontent.XContentBuilder;
+import org.elasticsearch.xcontent.XContentFactory;
+import org.elasticsearch.xcontent.XContentType;
+
+import java.io.IOException;
+import java.util.List;
+
+import static org.elasticsearch.common.xcontent.XContentHelper.stripWhitespace;
+import static org.hamcrest.Matchers.is;
+
+public class OpenShiftAIRerankRequestEntityTests extends ESTestCase {
+    private static final String INPUT = "documents";
+    private static final String QUERY = "query";
+    private static final String MODEL = "model";
```

Comment:
Minor point, but just to avoid the possibility of not catching weird bugs, it would be good to have these Strings not be equal to the field names they're associated with.